ONSdigital/eq-author-app
# Upgrade dependencies to fix issues building on node 10.

### What is the context of this PR?
Publisher did not seem to build correctly on a fresh install with node 10.
The issue was due to missing pre-compiled binaries for the fsevents package.
Performing a `yarn upgrade` fixed the dependencies in the `yarn.lock` file.

### How to review 
- Delete node_modules.
- Ensure you're using Node 10
- Run yarn install and ensure publisher is built correctly.
